using_redcap_api/using_redcap_api.md

```diff
@@ -0,0 +1,436 @@
+</div>
+
+Open a Jupyter notebook (running Python 3) and paste the code from the API Playground's "Python" tab into your first cell and run that cell.  It works!  Kind of.  Take a look below to see what the result of our API call for the records of cervical cancer data looks like.
+
+![The same python code as before, now in a Jupyter notebook](media/jupyter_1.png)<!-- style = "border :1px solid rgb(var(--color-highlight));" -->
+
+What's the only problem with this?  Well, it's not in a pandas data frame, which is what we'd prefer to work with.  We can fix that by adding a few lines of code!  Add this to a second cell, and run it.
+
+```python
+import pandas as pd
+from io import StringIO
+df = pd.read_csv(StringIO(r.text))
+df
+```
+
+Below, this is the result when we do that for our cervical cancer data database:
+
+![The python code listed above, and the resulting data frame](media/jupyter_2.png)<!-- style = "border :1px solid rgb(var(--color-highlight));" -->
+
+Now, your data is in a pandas data frame and you're ready to do all sorts of work with that data (like cleaning question marks out of the data, getting summary statistics, finding correlations, visualizing data, and modeling data).  Importantly, you can re-run the cells in this notebook at any time to get the latest version of the data from REDCap!  
+
+## R and the REDCap API
+
+REDCap assumes you're going to use an R script and not an R Markdown document, but we advocate using R Markdown for just about everything you do in R, so we're going to teach that approach.
+
+<div class = "important">
+There are useful R packages that make importing REDCap data into R simpler than the code REDCap suggests.  Nevertheless, we think it's important to show you how to use the code REDCap provides, because it is reliable and reflects the current state of the REDCap API.  
+
+The R packages that streamline this effort (we like REDCapR and redcapAPI) are great and seem very stable, and we provide links to them toward the end of this module, but R packages depend on volunteers who keep them updated.  Therefore, we think it's useful to know how to use the basic method we'll share here, even if in the future you prefer a different approach.
+</div>
+
+First, open RStudio and create a new R Markdown (File > New File > R Markdown).  Give your R Markdown the title and author information you want, leave the default output choice (HTML), and click on "OK".
+
+![The new R Markdown dialog box from RStudio](media/new_r_markdown.png)<!-- style = "border :1px solid rgb(var(--color-highlight));" -->
+
+Remove all the "boilerplate" text after the setup chunk (so, delete all the lines after around line 10) and add a new empty chunk:
+
+![R Markdown document with an empty chunk](media/empty_chunk.png)<!-- style = "border :1px solid rgb(var(--color-highlight));" -->
```

Comment:
This is similarly not giving the desired output:

<img width="633" alt="Screen Shot 2022-04-19 at 4 37 39 PM" src="https://user-images.githubusercontent.com/99294374/164091517-8249fa60-f251-4796-a50e-51687ab07b36.png">

Reply:
hmmm I wonder if the suggested code changed in the new version of REDCap??? I will do some more investigation here and in the python part

Reply:
fixed, made the instructions more clear